### PR TITLE
AAC fix port

### PR DESCRIPTION
**Pull request by TheNeoGamer42**
## About The Pull Request
this PR has caused me much pain, please help, is a port of https://github.com/yogstation13/Yogstation/pull/8281
## Why It's Good For The Game
mark said to port this
## Changelog
:cl:
fix: probably(tm) fixed the AAC making the game go insane about you doing too many clicks
/:cl:

### CODE REVIEW

File: tgui-next/packages/tgui/public/tgui.bundle.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
...
 /*! loadCSS. [c]2017 Filament Group, Inc. MIT License */
+<<<<<<< HEAD
```
Comment (Shadowtail117) [inline tgui-next/packages/tgui/public/tgui.bundle.js:3]: I don't think this should be in this file.

File: tgui-next/packages/tgui/public/tgui.bundle.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
...){return{id:e,name:(0,a.toTitleCase)(e.replace(/_/," ")),volume:l.recordingRecipe[e]}}))||l.beakerContents||[];return(0,o.createFragment)([(0,o.createComponentVNode)(2,c.Section,{title:"Status",buttons:u&&(0,o.createComponentVNode)(2,c.Box,{inline:!0,mx:1,color:"red",children:[(0,o.createComponentVNode)(2,c.Icon,{name:"circle",mr:1}),"Recording"]}),children:(0,o.createComponentVNode)(2,c.LabeledList,{children:(0,o.createComponentVNode)(2,c.LabeledList.Item,{label:"Energy",children:(0,o.createComponentVNode)(2,c.ProgressBar,{value:l.energy/l.maxEnergy,content:(0,r.toFixed)(l.energy)+" units"})})})}),(0,o.createComponentVNode)(2,c.Section,{title:"Recipes",buttons:(0,o.createFragment)([!u&&(0,o.createComponentVNode)(2,c.Box,{inline:!0,mx:1,children:(0,o.createComponentVNode)(2,c.Button,{color:"transparent",content:"Clear recipes",onClick:function(){return n("clear_recipes")}})}),!u&&(0,o.createComponentVNode)(2,c.Button,{icon:"circle",disabled:!l.isBeakerLoaded,content:"Record",onClick:function(){return n("record_recipe")}}),u&&(0,o.createComponentVNode)(2,c.Button,{icon:"ban",color:"transparent",content:"Discard",onClick:function(){return n("cancel_recording")}}),u&&(0,o.createCompon
```
Comment (Shadowtail117) [inline tgui-next/packages/tgui/public/tgui.bundle.js:7]: Or this?

**Review by Shadowtail117 — COMMENTED** [review]
Looks like you might have incorrectly tried to fix a merge conflict in AdvancedAirlockController.js.

**Comment by TheNeoGamer42** [comment]
yeah I fucking botched this anyways and my shit refused to work, not gonna bother, might redo it later though

**Pull request closed without merging**